deps/hiredis/COPYING

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2009-2011, Salvatore Sanfilippo <antirez at gmail dot com>
+Copyright (c) 2009-2011, Redis Ltd.
```

Comment:
FYI, this code was also vendored in hiredis, so will also be present in libvalkey https://github.com/valkey-io/libvalkey/blob/main/libvalkey/COPYING. Not sure if you also want to update that.